runtime.txt

```diff
@@ -1 +1 @@
-3.8
+python 3.8
```

Comment:
@ArshErgon I think Netlify requires this file only containing a version number: https://docs.netlify.com/configure-builds/manage-dependencies/#python

Reply:
@hudeven Sorry it's my first time, I have updated the readme.txt